Dockerfile

```diff
@@ -4,12 +4,15 @@ RUN apk add --update \
     && apk add --no-cache git\
     && rm -rf /var/cache/apk/*
 
-RUN git clone https://github.com/CSCfi/metadata-submitter-frontend.git
+RUN git clone -b develop https://github.com/CSCfi/metadata-submitter-frontend.git
```

Comment:
```suggestion
RUN git clone https://github.com/CSCfi/metadata-submitter-frontend.git
```

Reply:
that is ok. i will revert that change when metadata-front-end gets merged.

Reply:
used a smarter way to detect target branch https://github.com/CSCfi/metadata-submitter/pull/206/commits/999ce706d8693c17fa00b9567590e64d33d8d6a1